tests/apps_test.go

```diff
@@ -177,11 +178,24 @@ var _ = Describe("deis apps", func() {
 				Eventually(sess).Should(Exit(0))
 			})
 
+			//: INFO config %s updated
```

Comment:
Are all the commented lines being kept when/if getting the timestamp regex to work is revisited?  If so, could an issue tracking this be created and a corresponding comment be added here?